waku/factory/external_config.nim

```diff
@@ -312,8 +312,13 @@ type WakuNodeConf* = object
       name: "keep-alive"
     .}: bool
 
+    networkShards* {.
+      desc: "Number of shards in the network", defaultValue: 0, name: "network-shards"
+    .}: uint32
```

Comment:
:thinking: maybe we should be explicit that it pertains to autosharding?

What about `autoshardingRange` or `autoshardingsValues` ?

Reply:
Good question, not sure what should be the best name for it. I wouldn't personally use the word autosharding as even though we use this value for autosharding, it represents a property of the network. But open for opinions :))